File: spigot/core/src/main/java/com/wolfyscript/utilities/bukkit/gui/example/TestGUI.java
package com.wolfyscript.utilities.bukkit.gui.example;

import com.wolfyscript.utilities.WolfyCore;
import com.wolfyscript.viewportl.gui.GuiAPIManager;
import com.wolfyscript.viewportl.gui.example.CounterExampleKotlinKt;
import com.wolfyscript.viewportl.gui.example.StackEditorExampleKotlinKt;

public class TestGUI {

    private final WolfyCore core;

    public TestGUI(WolfyCore core) {
        this.core = core;
    }

    public void initWithConfig() {
        GuiAPIManager manager = core.getWolfyUtils().getGuiManager();
        CounterExampleKotlinKt.registerExampleCounter(manager);
        StackEditorExampleKotlinKt.registerStackEditor(manager);
    }


}

File: spigot/core/src/main/java/com/wolfyscript/utilities/bukkit/gui/example/TestGUI.kt
package com.wolfyscript.utilities.bukkit.gui.example

import com.wolfyscript.utilities.WolfyCore
import com.wolfyscript.viewportl.gui.example.CounterExampleKotlin
import com.wolfyscript.viewportl.gui.example.StackEditorExampleKotlin
import com.wolfyscript.viewportl.gui.example.StackSlotsExampleKotlin

class TestGUI(private val core: WolfyCore) {

    fun initWithConfig() {
        val manager = core.wolfyUtils.guiManager
        CounterExampleKotlin.registerExampleCounter(manager)
        StackEditorExampleKotlin.registerStackEditor(manager)
        StackSlotsExampleKotlin.registerStackSlotsExample(manager)
    }

}
